spec/rubocop/cop/rails/add_column_comment_spec.rb

```diff
@@ -0,0 +1,224 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::AddColumnComment, :config do
+  context 'when send add_column' do
+    it 'registers an offense when call has no `comment` option' do
+      expect_offense(<<~RUBY)
+        add_column :table, :column, :integer
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `add_column` without the `comment` option.
+      RUBY
+    end
+
+    it 'does not register an offense when call has `comment` option' do
+      expect_no_offenses(<<~RUBY)
+        add_column :table, :column, :integer, null: false, comment: 'An integer field', default: 0
+      RUBY
+    end
+  end
+
+  context 'when send create_table' do
+    it 'registers an offense when an `t.column` call has no `comment` option' do
```

Comment:
```suggestion
    it 'registers an offense when a `t.column` call has no `comment` option' do
```
(and similarly for the other examples)